### ISSUE TITLE
Please add support for starting a release from a tag

### ISSUE THREAD

**Issue by fields** [petervanderdoes/gitflow-avh#173]
I'm not sure if this is the right place for this - please let me know if not. This issue has manifested using Git Tower, and they've said they just pass the commands through to the AVH fork for git-flow. What I'm describing below used to work, and I don't know if it's because it's not supported in the AVH fork and they just switched to that, or something changed and it became no longer supported.

We use the develop branch as the deployment source for our staging environment, and set a tag when deployments are pushed to the staging server. We've been using those tags as the starting point for each release which is then deployed to the production environment. Tagging the deployment to staging and basing the release on that gives us assurance that the code in the release is exactly the same as the code we've tested in the staging environment.

As it seems to exist now in this version, a branch is required to start a release, which means if we want to keep using this workflow we'll have to create a new branch for every staging release and keep them around indefinitely, which would be messy. Just using the develop branch as the source for releases is problematic - people sometimes push changes back to the develop branch after the staging deployment but before the release is created. 

Please consider adding (or adding back) the ability to start a release from a tag.

Thank you!

**Comment by fields** [comment]
Any further thoughts?

**Comment by petervanderdoes** [comment]
Oh I probably will do this but maybe with an configuration to be set. something like gitflow.release.allow.from.tag or something.

Would  that work for you?

**Comment by fields** [comment]
That sounds fine to me, but I use this primarily through Tower.app, so I've asked them to weigh in as well.

**Comment by aiman-alsari** [comment]
Hi, I've just bumped into the same issue - git flow release start <release_name> <base> *always* starts the release from HEAD of <base>. In some cases it would be more suitable to provide a tag or a commit hash to release from.

Interestingly, this only seems to be an issue with this fork (which I'm only using because it comes with Ubuntu). The original nvie repo, and all documentation around it, implies that a commit hash/tag is used for <base> not a branch name.

**Comment by petervanderdoes** [comment]
Been short on time for a while, I hope to get to this in a few days.

**Comment by dmitryd** [comment]
@aiman-alsari 
> Interestingly, this only seems to be an issue with this fork (which I'm only using because it comes with Ubuntu). The original nvie repo, and all documentation around it, implies that a commit hash/tag is used for <base> not a branch name.

True. I use nvie's version locally and I have Peter's version at work. Local version works, at work it does not.

**Comment by dmitryd** [comment]
Oh, and it is really helpful to make a release from a tag or commit id. We generally need to make it from the commit id because this is what we have on stage for the customer to test. So I have to keep a copy of the git-flow where it works.

**Comment by stefancarlton** [comment]
I've just ran into this problem to. I resolved it by creating a temporary local branch and then releasing from there, which whilst messy, works. For future me...

```
git branch temp_branch <SHA1>
git flow release start <release_name> temp_branch
git branch -d temp_branch
```

**Comment by jbotuckBH** [comment]
I just ran into this problem as well. My workaround was to create the release branch with the right naming conventions manually. git flow release finish RELEASE worked as expected afterwards

**Comment by dmitryd** [comment]
Since I was waiting for too long on this to be implemented, I decided to do it without git flow. It is easy to do it with plain git. Not need for a wrapper like git flow anymore.
